docs/guide/en/configuring-rules-via-php-attributes.md

```diff
@@ -91,6 +91,22 @@ final class User
 }
 ```
 
+Alternatively, you can specify a label for all class properties:
+
+```php
+use Yiisoft\Validator\Label;
+use Yiisoft\Validator\Rule\Length;
+use Yiisoft\Validator\Rule\Required;
+
+#[Label('This value')]
```

Comment:
Why? What's the use-case for that?

Reply:
Like "**The value** should be 42"?

Reply:
"The value should be 42" by default. 
If a class or property has #[Label], then the names of attributes are used

https://github.com/yiisoft/validator/blob/1e73321d83d197b0effb16e5d2c32a95d60913e4/tests/ValidatorTest.php#L1464-L1509